pkg/fqdn/config.go

```diff
@@ -19,6 +19,10 @@ type Config struct {
 	// MinTTL is the time used by the poller to cache information.
 	MinTTL int
 
+	// Active Connections TTL is the time used by the GC to determines how long that
+	// connections should be idle for after the DNS TTL expires.
+	ActiveConnectionsTTL int
```

Comment:
What are the intended units for this integer value?

I think that we should probably be trying to store & use native time types for these configurations rather than storing integers and doing the type conversions on the fly. Unless there's another reason that makes this awkward, what do you think about specifying it with the following?

```suggestion
	// Active Connections TTL is the time used by the GC to determines how long that
	// connections should be idle for after the DNS TTL expires.
	ActiveConnectionsTTL time.Duration
```

Note that if we _do_ specify this with a native time type then we'll need to be a little more careful in how this is initialized from the user-provided argument, for instance if the flag is in seconds then multiply the user configuration by `time.Second`.

(In theory the same feedback should apply to `MinTTL` too, but that code is already there so I don't expect you to change it in this PR since it's unrelated. We can always iterate on that separately/after).

Reply:
I think specifying the `time.Duration` make sense to me as well. Need to check if there are any cases where this might cause problems